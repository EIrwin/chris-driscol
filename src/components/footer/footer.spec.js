// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import Component from './footer.component';

const fixtures = {
  social: {
    email: 'email',
    linkedIn: 'linkedIn',
    github: 'github',
  },
};

it('renders correctly', () => {
  const tree = renderer.create(<Component {...fixtures} />).toJSON();
  expect(tree).toMatchSnapshot();
});
