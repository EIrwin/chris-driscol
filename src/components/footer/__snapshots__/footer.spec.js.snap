// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-4"
      >
        <span
          className="copyright"
        >
          Copyright Â© 
          <strong>
            Chris Driscol
          </strong>
           
          2018
        </span>
      </div>
      <div
        className="col-md-4"
      >
        <ul
          className="list-inline social-buttons"
        >
          <li>
            <a
              href="linkedIn"
              target="_blank"
            >
              <svg
                fill="currentColor"
                height="1em"
                preserveAspectRatio="xMidYMid meet"
                style={
                  Object {
                    "color": undefined,
                    "verticalAlign": "middle",
                  }
                }
                viewBox="0 0 40 40"
                width="1em"
              >
                <g>
                  <path
                    d="m10.8 13.9v22.2h-7.4v-22.1h7.4z m0.5-6.8q0 1.7-1.2 2.7t-3 1.1h0q-1.9 0-3-1.1t-1.1-2.7q0-1.6 1.2-2.7t3-1.1 2.9 1.1 1.2 2.7z m26 16.3v12.7h-7.4v-11.9q0-2.3-0.9-3.6t-2.8-1.4q-1.4 0-2.3 0.8t-1.5 1.9q-0.2 0.7-0.2 1.8v12.4h-7.3q0-8.9 0-14.5t0-6.6l0-1h7.3v3.2h-0.1q0.5-0.7 1-1.3t1.2-1.1 2-1 2.5-0.4q3.8 0 6.2 2.6t2.3 7.4z"
                  />
                </g>
              </svg>
            </a>
          </li>
          <li>
            <a
              href="github"
              target="_blank"
            >
              <svg
                fill="currentColor"
                height="1em"
                preserveAspectRatio="xMidYMid meet"
                style={
                  Object {
                    "color": undefined,
                    "verticalAlign": "middle",
                  }
                }
                viewBox="0 0 40 40"
                width="1em"
              >
                <g>
                  <path
                    d="m15.8 27.1q0 0.9-0.3 1.9t-1 1.7-1.6 0.7-1.6-0.7-1-1.7-0.2-1.9 0.3-1.8 0.9-1.7 1.6-0.7 1.6 0.7 1 1.7 0.3 1.8z m14.3 0q0 0.9-0.3 1.9t-1 1.7-1.6 0.7-1.6-0.7-1-1.7-0.2-1.9 0.2-1.8 1-1.7 1.6-0.7 1.6 0.7 1 1.7 0.3 1.8z m3.5 0q0-2.6-1.5-4.5t-4.2-1.9q-0.9 0-4.3 0.5-1.6 0.2-3.5 0.2t-3.5-0.2q-3.4-0.5-4.4-0.5-2.6 0-4.2 1.9t-1.5 4.5q0 2 0.7 3.5t1.8 2.3 2.7 1.3 3.2 0.7 3.3 0.1h3.7q1.9 0 3.4-0.1t3.1-0.7 2.7-1.3 1.8-2.3 0.7-3.5z m5-3.9q0 4.6-1.3 7.4-0.9 1.7-2.4 3t-3.1 1.9-3.8 1.1-3.8 0.4-3.8 0.1q-1.7 0-3.1 0t-3.3-0.3-3.4-0.7-3.1-1.1-2.7-1.8-1.9-2.6q-1.4-2.7-1.4-7.4 0-5.3 3-8.8-0.6-1.9-0.6-3.8 0-2.6 1.2-4.9 2.4 0 4.2 0.9t4.2 2.8q3.3-0.8 6.9-0.8 3.3 0 6.3 0.7 2.3-1.8 4.2-2.7t4.2-0.9q1.1 2.3 1.1 4.9 0 1.9-0.6 3.7 3 3.6 3 8.9z"
                  />
                </g>
              </svg>
            </a>
          </li>
        </ul>
      </div>
      <div
        className="col-md-4"
      >
        <ul
          className="list-inline quicklinks"
        >
          <li>
            <a
              href="mailto:email"
            >
              email
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
</footer>
`;
